src/app/app.module.ts

```diff
@@ -158,18 +158,18 @@ export function kcFactory(keycloakService: KeycloakService) {
     SurveyBuilderService,
     UserService,
   ],
-  entryComponents: [
-    AddEditContactComponent,
-    AddEditOrganizationComponent,
-    LinkOrganizationComponent,
-    ContactSelectComponent,
-    ConfirmComponent,
-    InputModalComponent,
-    OrganizationsTableRowsComponent,
-    LinkOrganizationTableRowsComponent,
-    ContactSelectTableRowsComponent,
-    UserTableRowsComponent
-  ],
+  // entryComponents: [
```

Comment:
If entryComponents is deprecated as of Angular 9, this should be deleted instead of commented out